packages/git/src/browser/git-quick-open-service.ts

```diff
@@ -397,29 +424,19 @@ export class GitQuickOpenService {
     }
 
     async applyStash(): Promise<void> {
-        this.doStashAction('apply', 'Select a stash to \'apply\'.');
+        this.doStashAction('apply', nls.localizeByDefault('Pick a stash to apply'));
     }
 
     async popStash(): Promise<void> {
-        this.doStashAction('pop', 'Select a stash to \'pop\'.');
+        this.doStashAction('pop', nls.localizeByDefault('Pick a stash to pop'));
     }
 
     async dropStash(): Promise<void> {
         const repository = this.getRepository();
         if (!repository) {
             return;
         }
-        this.doStashAction('drop', 'Select a stash entry to remove it from the list of stash entries.',
-            async () => {
-                const list = await this.git.stash(repository, { action: 'list' });
-                let listString = '';
-                list.forEach(stashEntry => {
-                    listString += stashEntry.message + '\n';
-                });
-                return `Stash successfully removed.
-                There ${list.length === 1 ? 'is' : 'are'} ${list.length || 'no'} more entry in stash list.
-                \n${listString}`;
-            });
+        this.doStashAction('drop', nls.localizeByDefault('Pick a stash to drop'), async () => nls.localizeByDefault('theia/git/dropStashMessage', 'Stash successfully removed.'));
```

Comment:
I simplified output here because it is hard to localize messages in this format and because vscode does not output list of stashes after stash was dropped